src/main/java/ca/warp7/frc2020/subsystems/ControlPanelSpinner.java

```diff
@@ -26,46 +31,62 @@
 
     private static ControlPanelSpinner instance;
 
+    //These are calibration vars, if you're getting too high or
+    //too low a RGB value then what it should be outputting you
+    //can use these to make adustments to the input.
+    private final int redMod = 0;
+    private final int greenMod = -61;
```

Comment:
More documentation links, I think

The light sensor:  http://www.revrobotics.com/rev-31-1557/
And it has a datasheet  http://www.revrobotics.com/content/docs/REV-31-1557-DS.pdf
And the datasheet has a datasheet  https://docs.broadcom.com/docs/APDS-9151-DS

Nothing jumps out about why you needed to adjust green so much.  Weird.